src/OVAL/oval_filter.c

```diff
@@ -35,8 +35,8 @@
 #include <string.h>
 #include <stdlib.h>
 
-#include "common/debug_priv.h"
-#include "common/elements.h"
+#include <common/debug_priv.h>
+#include <common/elements.h>
```

Comment:
the convention is to use "" if you are including from openscap and <> if you are including from system.